### DISCUSSION TITLE
Unable to Use PlantUML in draw.io

### DISCUSSION THREAD

**Discussion by akonitx** [jgraph/drawio#3676] [Q&A] [answered]
I have `v24.1.0 Mac Universal Build` and from [this official guide from draw.io blog](https://www.drawio.com/blog/sketch-online-whiteboard) I see that i can use `PlantUML` but in my case in the `Advanced` panel there isn't `PlantUML` option, but in web (browser) version all good? 
How to fix this ?

**Comment by davidjgraph** [comment, marked as answer]
There is no mention of PlantUML in that post. The desktop project is here https://github.com/jgraph/drawio-desktop. The question has been asked previously, https://github.com/jgraph/drawio-desktop/issues?q=plantuml.
